### PR TITLE
chore: Move path_util dependency to util project

### PR DESCRIPTION
**Pull request by ghislainbourgeois**
Move `path_util` dependency to `util` project. Because it requires a bump of the `util_3gpp` that contained a fix to `suci.ToSupi()`, I also needed to fix the code calling that function, passing the parameters for both Profiles A and B, as defined in `TS 33.501`. The previous behavior always assumed Profile A.

This was tested with a successful simulation with `gnbsim`.

### CODE REVIEW

**Comment by ghislainbourgeois** [comment]
I found this commit in the upstream free5gc project that might be more appropriate: https://github.com/free5gc/udm/commit/f87f31e4616b17d2af16751421f3557b78da994d

@ajaythakurintel should I backport that instead?

**Review by thakurajayL — DISMISSED** [review]

**Comment by gab-arrobo** [comment]
> Yes..if you find appropriate fix then please port it.

@ghislainbourgeois please let me know how to proceed about this PR?

**Comment by ghislainbourgeois** [comment]
@gab-arrobo I am moving this PR back to draft and will port the change from upstream later today and tomorrow.

**Comment by ghislainbourgeois** [comment]
I back ported part of the fix from upstream, but left out the changes to the configuration, as it would require changing the configuration files in the helm charts and would break backwards compatibility. I tested again the latest changes through `gnbsim`. @gab-arrobo @thakurajayL

File: util/init_context.go [hunk 1]
```diff
@@ -53,7 +54,18 @@ func InitUDMContext(udmContext *context.UDMContext) {
 	udmContext.NrfUri = configuration.NrfUri
 	servingNameList := configuration.ServiceNameList
 
-	udmContext.Keys = configuration.Keys
+	udmContext.SuciProfiles = []suci.SuciProfile{
+		{
+			ProtectionScheme: "1",
```
Comment (gab-arrobo) [inline util/init_context.go:59]: Is it worth mentioning why `ProtectionScheme: "1"` (standard defined value for `Profile <A>`)? Or add a reference to TS 33.501 Annex C.1?

File: util/init_context.go [hunk 2]
```diff
@@ -53,7 +54,18 @@ func InitUDMContext(udmContext *context.UDMContext) {
...
+			PrivateKey:       configuration.Keys.UdmProfileAHNPrivateKey,
+			PublicKey:        configuration.Keys.UdmProfileAHNPublicKey,
+		},
+		{
+			ProtectionScheme: "2",
```
Comment (gab-arrobo) [inline util/init_context.go:64]: Is it worth mentioning why `ProtectionScheme: "2"` (standard defined value for `Profile <B>`)? Or add a reference to TS 33.501 Annex C.1?

**Review by gab-arrobo — APPROVED** [review]
+1 (LGTM, also tested changes with OnRamp)

File: util/init_context.go (hunk 1 above)
Comment (ghislainbourgeois) [reply util/init_context.go]: Agreed, I added a comment, thanks!

**Review by gab-arrobo — APPROVED** [review]
+1

**Pull request merged**